staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -213,12 +220,46 @@ func (s *EtcdOptions) Complete(storageObjectCountTracker flowcontrolrequest.Stor
 	}
 
 	if len(s.EncryptionProviderConfigFilepath) != 0 {
-		transformerOverrides, kmsPluginHealthzChecks, err := encryptionconfig.LoadEncryptionConfig(s.EncryptionProviderConfigFilepath, stopCh)
+		ctx, closeTransformers := wait.ContextForChannel(stopCh)
+		transformerOverrides, kmsPluginHealthzChecks, configContentHash, _, err := encryptionconfig.LoadEncryptionConfig(s.EncryptionProviderConfigFilepath, ctx.Done())
```

Comment:
Timeout needs to be passed to `NewDynamicTransformers`

```suggestion
		transformerOverrides, kmsPluginHealthzChecks, configContentHash, timeout, err := encryptionconfig.LoadEncryptionConfig(s.EncryptionProviderConfigFilepath, ctx.Done())
```